compiler/cli/src/org/jetbrains/kotlin/cli/metadata/K2MetadataCompiler.kt

```diff
@@ -53,6 +54,9 @@ class K2MetadataCompiler : CLICompiler<K2MetadataCompilerArguments>() {
     ): ExitCode {
         val collector = configuration.getNotNull(CLIConfigurationKeys.MESSAGE_COLLECTOR_KEY)
 
+        val pluginLoadResult = PluginCliParser.loadPluginsSafe(arguments, configuration)
+        if (pluginLoadResult != ExitCode.OK) return pluginLoadResult
```

Comment:
I think would be better to report something here.

cc @udalov

Reply:
Some errors reported in `loadPluginsSafe` function:

https://github.com/JetBrains/kotlin/blob/master/compiler/cli/src/org/jetbrains/kotlin/cli/jvm/plugins/PluginCliParser.kt#L43

Reply:
Looks fine to me

Reply:
ok